plynx/web/validation.py

```diff
@@ -0,0 +1,43 @@
+import re
+from plynx.db.user import User
+
+def validate_user(**kwargs):
```

Comment:
can this function be in `plynx/plynx/db/user` as a method of the class? Also I don't think using kwargs is justified here